PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/PBMCore/NativeAssets/NativeAsset.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  NativeAsset.swift
+//  PrebidMobileRendering
+//
+//  Copyright © 2021 Prebid. All rights reserved.
+//
+
+import Foundation
+
+public class NativeAsset: NSObject, NSCopying, PBMJsonCodable {
+    
+    /// [Required]
+    /// [Integer]
+    /// Unique asset ID, assigned by exchange. Typically a counter for the array.
+    /// Assigned by SDK.
+    @objc public var assetID: NSNumber?
+    
+    /// [Integer]
+    /// Set to 1 if asset is required (exchange will not accept a bid without it)
+    @objc public var required: NSNumber?
+    
+    /// This object is a placeholder that may contain custom JSON agreed to by the parties
+    /// to support flexibility beyond the standard defined in this specification
+    @objc private(set) public var assetExt: [String : Any]?
+    
+    @objc private(set) public var childExt: [String : Any]?
+    
+    var childType: String
+    
+    private override init() {
+        NSLog("NativeAsset class should not be instantialted directly, instantiate subclasses instead.")
```

Comment:
Need to remove NSLog and use internal logging system

Reply:
Done